custom/icds_reports/utils/fix_child_health_missing_month.sql

```diff
@@ -0,0 +1,613 @@
+--                                  Hash Cond: (agg_child.awc_id = ucr.doc_id)
+--                                  ->  Seq Scan on "agg_child_health_2018-05-01_5" agg_child  (cost=0.00..427238.73 rows=4579073 width=224)
+--                                  ->  Hash  (cost=119279.11..119279.11 rows=612805 width=31)
+--                                        ->  Unique  (cost=0.42..113151.06 rows=612805 width=31)
+--                                              ->  Index Scan using awc_location_local_doc_id_idx on awc_location_local ucr  (cost=0.42..111331.32 rows=727897 width=31)
+--                                                    Filter: ((awc_is_test = 0) AND (aggregation_level = 5))
+--          ->  Hash  (cost=8827.09..8827.09 rows=93809 width=355)
+--                ->  Seq Scan on "agg_child_health_2018-05-01_4" agg_child_health  (cost=0.00..8827.09 rows=93809 width=355)
+
+
+UPDATE "agg_child_health_2018-05-01_3" agg_child_health
+    SET wasting_moderate = ut.wasting_moderate,
+        wasting_severe = ut.wasting_severe,
+        stunting_moderate = ut.stunting_severe,
+        stunting_severe = ut.stunting_severe,
+        wasting_normal = ut.wasting_normal,
+        stunting_normal = ut.stunting_normal,
+        wasting_normal_v2 = ut.wasting_normal_v2,
+        wasting_moderate_v2 = ut.wasting_moderate_v2,
+        wasting_severe_v2 = ut.wasting_severe_v2,
+        zscore_grading_hfa_normal = ut.zscore_grading_hfa_normal,
+				zscore_grading_hfa_moderate = ut.zscore_grading_hfa_moderate,
+				zscore_grading_hfa_severe = ut.zscore_grading_hfa_severe,
+				zscore_grading_hfa_recorded_in_month = ut.zscore_grading_hfa_recorded_in_month,
+				zscore_grading_wfh_recorded_in_month = ut.zscore_grading_wfh_recorded_in_month
+    from (
+        SELECT
+        block_id,
+        gender,
+        age_tranche,
+        SUM(wasting_moderate) as wasting_moderate,
+        SUM(wasting_severe) as wasting_severe,
+        SUM(stunting_moderate) as stunting_moderate,
+        SUM(stunting_severe) as stunting_severe,
+        SUM(wasting_normal) as wasting_normal,
+        SUM(stunting_normal) as stunting_normal,
+        SUM(wasting_normal_v2) as wasting_normal_v2,
+        SUM(wasting_moderate_v2) as wasting_moderate_v2,
+        SUM(wasting_severe_v2) as wasting_severe_v2,
+        SUM(zscore_grading_hfa_normal) as zscore_grading_hfa_normal,
+				SUM(zscore_grading_hfa_moderate) as zscore_grading_hfa_moderate,
+				SUM(zscore_grading_hfa_severe) as zscore_grading_hfa_severe,
+				SUM(zscore_grading_hfa_recorded_in_month) as zscore_grading_hfa_recorded_in_month,
+				SUM(zscore_grading_wfh_recorded_in_month) as zscore_grading_wfh_recorded_in_month
+
+        FROM "agg_child_health_2018-05-01_4" agg_child INNER JOIN (SELECT DISTINCT ucr.supervisor_id FROM "awc_location_local" ucr WHERE ucr.supervisor_is_test=0 AND aggregation_level=4) tt ON tt.supervisor_id = agg_child.supervisor_id
```

Comment:
what are these inner joins for?

Reply:
@Rohit25negi Can you explain this, I just pick this one out from your sample script.

Reply:
hey, this is because in older months, there were test locations in non test states as well. So this is avoid aggregating test locations in real states.